specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -1289,6 +1294,231 @@ This header contains the Lifetime Management API
+    {
+        /// Initialize the calling process to use Windows App SDK's framework package.
+        ///
+        /// Find a Windows App SDK framework package meeting the criteria and make it available
+        /// for use by the current process. If multiple packages meet the criteria the best
+        /// candidate is selected.
+        ///
+        /// This is equivalent to `Initialize(majorMinorVersion, null, new PackageVersion())`.
+        ///
+        /// @param majorMinorVersion major and minor version of Windows App SDK's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+        /// @see Initialize(uint, string)
+        /// @see Initialize(uint, string, PackageVersion)
+        /// @see Shutdown()
+        public static int Initialize(uint majorMinorVersion)
+        {
+            return Initialize(majorMinorVersion, null);
+        }
+
+        /// Initialize the calling process to use Windows App SDK's framework package.
+        ///
+        /// Find a Windows App SDK framework package meeting the criteria and make it available
+        /// for use by the current process. If multiple packages meet the criteria the best
+        /// candidate is selected.
+        ///
+        /// This is equivalent to `Initialize(majorMinorVersion, versionTag, new PackageVersion())`.
+        ///
+        /// @param majorMinorVersion major and minor version of Windows App SDK's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+        /// @param versionTag version tag (if any), e.g. "preview1".
+        /// @see Initialize(uint)
+        /// @see Initialize(uint, string, PackageVersion)
+        /// @see Shutdown()
+        public static int Initialize(uint majorMinorVersion, string versionTag)
+        {
+            var minVersion = new PackageVersion();
+            return Initialize(majorMinorVersion, versionTag, minVersion);
+        }
+
+        /// Initialize the calling process to use Windows App SDK's framework package.
+        ///
+        /// Find a Windows App SDK framework package meeting the criteria and make it available
+        /// for use by the current process. If multiple packages meet the criteria the best
+        /// candidate is selected.
+        ///
+        /// @param majorMinorVersion major and minor version of Windows App SDK's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+        /// @param versionTag version tag (if any), e.g. "preview1".
+        /// @param minVersion the minimum version to use
+        /// @see Initialize(uint)
+        /// @see Initialize(uint, string)
+        /// @see Shutdown()
+        public static int Initialize(uint majorMinorVersion, string versionTag, PackageVersion minVersion)
+        {
+            return NativeMethods.MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion);
+        }
+
+        /// Undo the changes made by Initialize().
```

Comment:
I'd remove all this and link to your existing PR. Better to not ossify potentially-living code in documentation, it's code.

Reply:
Not following what you mean by "all this"

API Reviews are usually IDL and Flat-C headers, i.e. all interface. C# has doesn't do that last I looked. That's fine, the spec should be C#-like, just omit all bodies as N/A for specs+review? For instance this

```
        public static void Shutdown()
        {
            NativeMethods.MddBootstrapShutdown();
        }
```

should be spec'd as

```
        public static void Shutdown();
```

?

Reply:
Ah, yes.  Put the _interface_ here and leave the implementation in the .cs file, thanks.